buildenv/jenkins/docker-slaves/ppc64le/centos7/Dockerfile

```diff
@@ -0,0 +1,246 @@
+    cpio \
+    elfutils-libelf-devel \
+    file-devel \
+    file-libs \
+    flex \
+    fontconfig \
+    fontconfig-devel \
+    freetype-devel \
+    gcc-4.8.5 \
+    gcc-c++-4.8.5 \
+    gettext \
+    glibc \
+    glibc-common \
+    glibc-devel \
+    gmp-devel \
+    java-1.8.0-openjdk-devel \
+    lbzip2 \
+    libdwarf \
+    libstdc++-static \
+    libX11-devel \
+    libXext-devel \
+    libXrender-devel \
+    libXt-devel \
+    libXtst-devel \
+    mesa-libGL-devel \
+    mpfr-devel \
+    numactl-devel \
+    ntp \
+    openssl-devel \
+    openssh-server \
+    openssh-clients \
+    perl-CPAN \
+    perl-DBI \
+    perl-devel \
+    perl-GD \
+    perl-libwww-perl \
+    perl-Time-HiRes \
+    unzip \
+    vim \
+    wget\
+    xz \
+    zip \
+    libdwarf-devel \
+  && rm -rf /var/lib/apt/lists/*
+
+# Install autoconf version 2.69
+RUN cd /usr/src/ \
+  && wget https://fossies.org/linux/misc/autoconf-2.69.tar.gz \
+  && tar -xzf autoconf-2.69.tar.gz \
+  && rm -f autoconf-2.69.tar.gz \
+  && cd autoconf-2.69 \
+  && ./configure --build=$(rpm --eval %{_host}) \
+  && make \
+  && make install
+
+# Dependency required by test framework
+RUN wget -O - http://cpanmin.us | perl - --self-upgrade \
+  && cpanm Text::CSV \
+  && cpanm JSON
+
+# Setup links to GCC-4.8.5
+RUN rm /usr/bin/c++ /usr/bin/cc \
+  && ln -s g++ /usr/bin/c++ \
+  && ln -s gcc /usr/bin/cc
+
+# Install GCC-7.3.1
+RUN yum -y install environment-modules \
+  && source /etc/profile.d/modules.sh \
+  && wget ftp://ftp.unicamp.br/pub/linuxpatch/toolchain/at/redhat/RHEL7/gpg-pubkey-6976a827-5164221b \
+  && rpm --import gpg-pubkey-6976a827-5164221b \
+  && echo "# Begin of configuration file" >> /etc/yum.repos.d/at11.0.repo \
+  && echo "[at11.0]" >> /etc/yum.repos.d/at11.0.repo \
+  && echo "name=Advance Toolchain Unicamp FTP" >> /etc/yum.repos.d/at11.0.repo \
+  && echo "baseurl=ftp://ftp.unicamp.br/pub/linuxpatch/toolchain/at/redhat/RHEL7" >> /etc/yum.repos.d/at11.0.repo \
+  && echo "failovermethod=priority" >> /etc/yum.repos.d/at11.0.repo \
+  && echo "enabled=1" >> /etc/yum.repos.d/at11.0.repo \
+  && echo "gpgcheck=1" >> /etc/yum.repos.d/at11.0.repo \
+  && echo "gpgkey=ftp://ftp.unicamp.br/pub/linuxpatch/toolchain/at/redhat/RHEL7/gpg-pubkey-6976a827-5164221b" >> /etc/yum.repos.d/at11.0.repo \
+  && echo "# End of configuration file" >> /etc/yum.repos.d/at11.0.repo \
+  && yum -y install advance-toolchain-at11.0-runtime advance-toolchain-at11.0-devel advance-toolchain-at11.0-perf
+
+#Building and setting up git version 2.5.3
+RUN yum -y install curl-devel expat-devel gettext-devel openssl-devel zlib-devel perl-ExtUtils-MakeMaker \
+  && cd /usr/src \
+  && wget https://www.kernel.org/pub/software/scm/git/git-2.5.3.tar.gz \
+  && tar xzf git-2.5.3.tar.gz \
+  && rm git-2.5.3.tar.gz \
+  && cd git-2.5.3 \
+  && make prefix=/usr/local/git all \
+  && make prefix=/usr/local/git install \
+  && rm -f /usr/bin/git \
+  && ln -s /usr/local/git/bin/git /usr/bin/git
```

Comment:
Since we are installing to /usr/local I think this symlink is unnecessary.

Reply:
Technically it gets installed to /usr/local/git which isn't on the path at all, which makes the symlink necessary. Should I change the install location to /usr/local and remove the symlink, or is git required to be installed in this location?

Reply:
We should install it to `/usr/local` (`not /usr/local/git`) to avoid colliding with package managed git which installs to `/usr/bin/git`. This should avoid the need for a symlink.